ST_params.c

```diff
@@ -928,8 +859,6 @@ static void _rgroup_init( void) {
    }
    _rgroup_addsucculent( name, wslope, wint, dslope, dint);
 
-   /* ------------------------------------------------------------ */
-   /* Get wildfire parameters */
```

Comment:
Perhaps we should retain this comment?

Reply:
@fdpierson Definitely. Don't know when I removed this, but it might have been an accident.

Reply:
d426829 addresss this issue.

Reply:
@fdpierson not a problem and thanks!